vmm/src/memory_manager.rs

```diff
@@ -1621,12 +1617,20 @@ impl MemoryManager {
         match self.hotplug_method {
             HotplugMethod::VirtioMem => {
                 if desired_ram >= self.boot_ram {
+                    if !self.dynamic {
```

Comment:
This looks wrong as there is `>=` here ^

Reply:
Looking at the code I think this check should be `>` rather than `>=`, no?
With the suggested change, the `==` part will become a no-op there.

Reply:
@fidencio That was exactly my point. That check is wrong.